subsys/bluetooth/mesh/subnet.c

```diff
@@ -82,6 +82,9 @@ static void key_refresh(struct bt_mesh_subnet *sub, uint8_t new_phase)
 		break;
 	/* Revoking keys */
 	case BT_MESH_KR_PHASE_3:
+		if (sub->kr_phase == BT_MESH_KR_NORMAL) {
+			return;
+		}
```

Comment:
You'll need to add `__fallthrough;` after this so that it doesn't get flagged by static analysers (it's also a requirement of the coding guidelines)